src/panel/pages/timeline/components/TimelineTooltip.tsx

```diff
@@ -15,24 +15,37 @@ export interface TooltipPosition {
   y: number;
 }
 
-export const TimelineTooltip: FC<JSX.IntrinsicElements["div"]> = ({
-  children,
-  ...props
-}) => (
-  <div {...props} style={{ paddingBottom: 8, zIndex: 9, ...props.style }}>
-    <TooltipElement>{children}</TooltipElement>
-  </div>
-);
-
-const TooltipElement = styled.p<{ position?: TooltipPosition }>`
+type TimelineTooltipProps = {
+  handleTooltipRef?: (passedRef: any) => void;
+  tooltipOffset?: number;
+};
+
+export const TimelineTooltip: FC<
+  JSX.IntrinsicElements["div"] & TimelineTooltipProps
+> = ({ children, handleTooltipRef, tooltipOffset, ...props }) => {
+  const offset = useMemo(() => (tooltipOffset ? tooltipOffset : 0), [
+    tooltipOffset,
+  ]);
+  return (
+    <div
+      {...props}
+      style={{ marginLeft: tooltipOffset, ...props.style }}
+      ref={handleTooltipRef}
+    >
+      <TooltipElement offset={offset}>{children}</TooltipElement>
+    </div>
+  );
+};
+
+const TooltipElement = styled.p<{ position?: TooltipPosition; offset: number }>`
   position: relative;
   background-color: ${(p) => p.theme.dark["-3"]};
   border-radius: 2px;
   color: #fff;
   font-size: 12px;
   margin: 0;
   padding: 10px 20px;
-  z-index: 9;
```

Comment:
Good shout on removing this! Position fixed should already ensure we are on top of all other elements.